youths/views.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.http import HttpResponse, JsonResponse
+
+from youths.models import YouthProfile
+
+
+def profiles(request, *args, **kwargs):
+    if settings.GDPR_API_ENABLED:
+        # TODO: Add authentication and security
```

Comment:
I guess this endpoint would only be usable by the profile backend? No user client will access this endpoint i.e. adding `login_required` decorator is not what we want here?

Reply:
I guess we are going to implement some kind of client cert authentication or such. Still to be defined and until then we just disable it.